students/gangho/users/views.py

```diff
@@ -48,4 +48,25 @@ def post(self, request):
         except IntegrityError:
             return JsonResponse({
                 'message': 'email overlap.'
-            }, status=400)
\ No newline at end of file
+            }, status=400)
+
+class SigninView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email    = data['email']
+            password = data['password']
+
+            validate_email(email)
+            validate_password(password)
+
+            if User.objects.filter(email=email, password=password).exists():
+                return JsonResponse({'message': 'SUCCESS'}, status=200)
+
+            return JsonResponse({'message': 'INVALID_USER'}, status=401)
```

Comment:
`FAIL`에 해당하는 코드를 `if문`에 넣어서 예외 처리 하는 것이 더 가독성이 좋을 것 같습니다.
```suggestion
            if not User.objects.filter(email=email, password=password).exists():
                return JsonResponse({'message': 'INVALID_USER'}, status=401)

            return JsonResponse({'message': 'LOGIN_SUCCESS'}, status=200)
```

Reply:
수정해서 다시 올리도록 하겠습니다!